easybuild/easyconfigs/g/gaussian/gaussian-09.e.01-AVX.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
```

Comment:
The "permissions" step is run after any postinstallcmds and forcibly sets read access to everything that has been installed.

The problem with that is that the gaussian binary starts by checking that it as been installed with "other" unreadable.

And i did mention this a while ago and you did agree that we need a umask value in the easyconfig.